src/Appwrite/Platform/Modules/DevKeys/Http/DevKeys/UpdateKey.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Appwrite\Platform\Modules\DevKeys\Http\DevKeys;
+
+use Appwrite\Extend\Exception;
+use Appwrite\Utopia\Response;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+use Utopia\Database\Validator\Datetime as DatetimeValidator;
+use Utopia\Database\Validator\UID;
+use Utopia\Platform\Action;
+use Utopia\Platform\Scope\HTTP;
+use Utopia\Validator\Text;
+
+class UpdateKey extends Action
```

Comment:
Should probably be UpdateDevKey or UpdateProjectDevKey. Let's also verify with @Meldiron for the Sites PR to be consistent.